### PR TITLE
[Trello 107] Delete avatar

### PR DESCRIPTION
**Pull request by omelniz**

### CODE REVIEW

File: src/components/Settings/index.service.js [hunk 1]
```diff
@@ -10,26 +12,34 @@ const SettingsService = ({ children }) => {
   const dispatch = useDispatch()
   const navigation = useNavigation()
 
-  const authSignout = useSelector(state => state.auth.authSignout)
+  const errorMessage = useSelector(usersSelector.usersDeleteAvatarError)
```
Comment (azimgd) [inline src/components/Settings/index.service.js:15]: `errorMessage` variable here in Settings context is to ambiguous. Let's change it to `usersDeleteAvatar` selector and later access error message through `usersDeleteAvatar.error.text`

Comment (azimgd) [reply src/components/Settings/index.service.js:15]: This way you could also add Loading indicators

File: src/components/Settings/index.service.js [hunk 2]
```diff
@@ -10,26 +12,34 @@ const SettingsService = ({ children }) => {
...
-    navigationActions.navigateProfilePhotoUpload(navigation, ({ type: 'IMAGE', photos: [payload[0].preview] }))()
+    dispatch(cameraActions.cameraCaptureRequest(payload))
+    navigationActions.navigateProfilePhotoUpload(navigation, { type: 'IMAGE', photos: [payload[0].preview] })()
   }
 
   const camera = useCamera({
     handleProcessedPhoto,
   })
 
+  const deleteProfilePhoto = () => dispatch(usersActions.usersDeleteAvatarRequest())
```
Comment (azimgd) [inline src/components/Settings/index.service.js:30]: We have a convention of naming request variables in the same format as action. This way you could rename variable to `usersDeleteAvatarRequest`

**Review by azimgd — CHANGES_REQUESTED** [review]

File: src/components/Settings/index.service.js (hunk 1 above)
Comment (omelniz) [reply src/components/Settings/index.service.js]: Done

File: src/components/Settings/index.service.js (hunk 2 above)
Comment (omelniz) [reply src/components/Settings/index.service.js]: Done

File: src/components/Settings/index.service.js (hunk 1 above)
Comment (omelniz) [reply src/components/Settings/index.service.js]: I rename the errorMessage variable to settingsErrorMessage to make one common error for the settings screen similarly with formErrorMessage variable

**Pull request merged**